src/components/Search/Search.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { MoviesList } from '../MoviesList';
+
+interface Props {
+  movies: Movie[];
+}
+
+interface State {
+  query: string;
+}
+
+export class Search extends React.Component<Props, State> {
+  state: State = {
+    query: '',
+  };
+
+  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    this.setState({
+      query: value,
+    });
+  };
+
+  render() {
+    const { query } = this.state;
+    const { movies } = this.props;
```

Comment:
you don't use movies in this component, you just pass it on. So it's better to render both components (MoviesList and Search) in the App

Reply:
yes, I should use lifting state up or render components in the App